config/config.exs

```diff
@@ -3,7 +3,8 @@ import Config
 config :open_trip_planner_client,
   otp_url: "http://otp2-local.mbtace.com",
   # otp_url: "http://localhost:8080",
-  timezone: "America/New_York"
+  timezone: "America/New_York",
+  fallback_error_message: "Something else went wrong."
```

Comment:
What about just `"Something went wrong"`? If I'm reading the rest of this PR right, this error would appear in a context like:

```elixir
%OpenTripPlannerClient.Error{
  message: "Something else went wrong."
}
```

Which feels to me like it wants to be at the end of a long list of errors: "Your origin was too far away from anything, your destination was too warm, the bus driver on the optimal route doesn't like you, AND something else went wrong".

I'd go with either "Something else went wrong" or "Unknown error".